src/utils.ts

```diff
@@ -25,11 +25,11 @@ export const mapProperties = (
         else throw new Error(`Can't find model ${schema}`)
 
     return Object.entries(schema?.properties ?? []).map(([key, value]) => {
-        const { type: valueType = undefined, ...rest } = value as any
+        const { type: valueType = undefined, description, examples, ...schemaKeywords } = value as any
         return {
             // @ts-ignore
-            ...rest,
```

Comment:
do we need to keep this spread of rest at the root of the return and not in `schema`?

i'll be pulling and testing out tonight but wanted to give a review before that and ask ahead :)

Reply:
@marclave 
Had a thought of it and I think it makes more sense to be under `schema`. 

Most [JSON schema keywords](https://swagger.io/docs/specification/data-models/keywords/) goes under `schema` except for `description`, `required`, `examples`.

Also when we choose to define some keywords and send others to spread, it makes more sense to define generic keywords explicitly and send type-specific keywords like `minLength`, which belongs to `schema`, to the rest